src/app/components/Fragment/index.tsx

```diff
@@ -0,0 +1,41 @@
+/** @jsxRuntime classic */
+/** @jsx  jsx  */
+/* @jsxFrag React.Fragment */
+import React, { PropsWithChildren } from 'react';
+import { jsx } from '@emotion/react';
+import styles from './index.style';
+
+const bold = ({ children }: PropsWithChildren) => <b>{children}</b>;
+
+const italic = ({ children }: PropsWithChildren) => (
+  <i css={styles.italicStyle}>{children}</i>
```

Comment:
Similar here, could try the `<Text>` component to set the `as`, `size` and `fontVariant` props.